tests/Microsoft.Identity.Test.Unit/PublicApiTests/AuthenticationResultTests.cs

```diff
@@ -46,10 +46,69 @@ public void GetAuthorizationHeader()
                 "tid",
                 new Account("aid", "user", "env"),
                 "idt", new[] { "scope" }, Guid.NewGuid(),
-                new AuthenticationResultMetadata(TokenSource.Cache),
-                "SomeTokenType");
+                "SomeTokenType",
+                new AuthenticationResultMetadata(TokenSource.Cache));
 
             Assert.AreEqual("SomeTokenType at", ar.CreateAuthorizationHeader());
         }
+
+        [TestMethod]
+        [Description(
+            "In MSAL 4.17 we made a mistake and added AuthenticationResultMetadata with no default value before tokenType param. " +
+            "To fix this breaking change, we added 2 constructors - " +
+            "one for backwards compat with 4.17+ and one for 4.16 and below")]
```

Comment:
>4.16 and below [](start = 61, length = 14)

Additional comment: which is also the go forward recommended ctor, and way of initializing.
Similar meaning I would like to convey in the public API.